lsdpack.cpp

```diff
@@ -136,21 +136,23 @@ void make_out_path(const char* in_path, std::string suffix) {
     printf("Recording to '%s'\n", out_path.c_str());
 }
 
-void load_gb(const char* path) {
-    if (gameboy.load(path)) {
+void load_gb(const char* path, unsigned flags) {
```

Comment:
to make the code more readable, i think it would be preferable to pass in `bool dmg_mode` and have it converted to `unsigned` in call to `gameboy.load`.